### ISSUE TITLE
Write after end

### ISSUE THREAD

**Issue by yoshuawuyts** [stencila/sibyl#14]
Getting a pipe error in the server; suspect it's because of errors / unpipe events not handled correctly
```txt
  Error: write after end
      at writeAfterEnd (_stream_writable.js:220:12)
      at PassThrough.Writable.write (_stream_writable.js:271:5)
      at Socket.sibyl.stderr.on.data (/Users/anon/src/stencila/sibyl/server/server.js:49:11)
      at emitOne (events.js:115:13)
      at Socket.emit (events.js:210:7)
      at addChunk (_stream_readable.js:252:12)
      at readableAddChunk (_stream_readable.js:239:11)
      at Socket.Readable.push (_stream_readable.js:197:10)
      at Pipe.onread (net.js:588:20)
```

**Comment by yoshuawuyts** [comment]
Ah yeah, so I suspect `koa` is closing the `through` stream

**Comment by yoshuawuyts** [comment]
Closed by #18
